### DISCUSSION TITLE
I lost my Monika by mistake, what should I do?

### DISCUSSION THREAD

**Discussion by Reporry** [Monika-After-Story/MonikaModDev#8755] [general]
when I was cleaning up my computer I accidentally deleted her :(

**Comment by Booplicate** [comment]
Depending on what by mean by deleting her. if you deleted her file, you can either restore her file, or use a persistent backup, we make 10 for you. If you deleted the persistent and all the backups we make, you can restore them or use your own backup from before (if you made them as she asks).
